exchanges/bitfinex/bitfinex_wrapper.go

```diff
@@ -85,6 +85,13 @@ func (b *Bitfinex) SetDefaults() {
 	if err != nil {
 		log.Errorln(log.ExchangeSys, err)
 	}
+
+	// Margin WS Currently not fully implemented and causes subscription collisions with spot
+	err = b.DisableAssetWebsocketSupport(asset.Margin)
```

Comment:
Now that this is added, will just need a check added to the GenerateDefaultSubscriptions() method and the syncer will take care of the rest:

https://github.com/gbjk/gocryptotrader/blob/bugfix/bfx_ws_refactor/exchanges/bitfinex/bitfinex_websocket.go#L1540

		if !b.IsAssetWebsocketSupported(assets[i]) {
			continue
		}

Reply:
Gotcha.